File: Array/1099.Two Sum Less Than K/Solution.java
class Solution {
    public int twoSumLessThanK(int[] nums, int k) {
        int ans = -1, n = nums.length;
        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {
                if (nums[i] + nums[j] < k) {
                    ans = Math.max(ans, nums[i] + nums[j]);
                }
            }
        }
        return ans;
    }
}

File: Array/1099.Two Sum Less Than K/Solution.py
class Solution:
    def twoSumLessThanK(self, nums: List[int], k: int) -> int:
        ans, n = -1, len(nums)
        for i in range(n):
            for j in range(i + 1, n):
                if nums[i] + nums[j] < k:
                    ans = max(nums[i] + nums[j], ans)
        return ans